scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -261,7 +262,12 @@ def generate_command(self):
         handler: Handler = self.instance.handler
 
         if handler.options.verbose > 1:
-            command.append('--log-level=DEBUG')
+            command.extend([
+                '--log-level=DEBUG',
+                '-o', 'log_cli=true',
+                '--log-cli-format=%(levelname)s %(message)s',
+                f'--log-file={self.pytest_log_file_path}'
+            ])
 
```

Comment:
```suggestion
log_cli_level = 'WARNING'
if handler.options.verbose == 1:
    log_cli_level = 'INFO'
elif if handler.options.verbose > 1:
    log_cli_level = 'DEBUG'
    
  command.extend([
      '--log-file-level=DEBUG',
      '-o', 'log_cli=true',
      '--log-cli-format=%(levelname)s %(message)s',
      f'--log-file={self.pytest_log_file_path}',
      f'--log-cli-level={log_cli_level}'
  ])
```

logowanie do pliku chcemy mieć zawsze, niezależnie od `verbose`. A od `verbose` możemy uzależnić printowanie na konsole

Reply:
trzeba potestować .. bo jeśli `not verbose`, to też nie powinniśmy dodawać `-o log_cli=true`